presto-main/src/main/java/com/facebook/presto/operator/scalar/FnvHash.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.scalar;
+
+import io.airlift.slice.Slice;
+
+// Taken from https://github.com/airlift/slice/blob/95d5876d3f4c57321767113e97416a42bc41749a/src/main/java/io/airlift/slice/FnvHash.java
+class FnvHash
+{
+    private static final long FNV_64_OFFSET_BASIS = 0xcbf29ce484222325L;
+    private static final long FNV_64_PRIME = 0x100000001b3L;
+
+    private static final int FNV_32_OFFSET_BASIS = 0x811c9dc5;
+    private static final int FNV_32_PRIME = 0x01000193;
+
+    private FnvHash()
+    {
+    }
+
+    static int fnv1_32(Slice data)
```

Comment:
Presto Java code follow camel convention (and don't usually use `_`). What about `fnv1Hash32`? ? Ditto for other methods.

Reply:
can this be a public method?